src/Api/Controllers/AccountsController.cs

```diff
@@ -534,6 +539,11 @@ public async Task PostPayment([FromBody]PaymentRequestModel model)
                 throw new UnauthorizedAccessException();
             }
 
+            await _paymentService.SaveTaxInfoAsync(user, new TaxInfo
+            {
+                BillingAddressCountry = model.Country,
+                BillingAddressPostalCode = model.PostalCode,
+            });
             await _userService.ReplacePaymentMethodAsync(user, model.PaymentToken, model.PaymentMethodType.Value);
```

Comment:
Why not treat this like ReplacePaymentMethodAsync for org? Pass in TaxInfo and handle this in the service.

Reply:
Fair point, I'll make that change.